src/Adapter/Order/OrderSiblingProvider.php

```diff
@@ -0,0 +1,29 @@
+<?php
+
+namespace PrestaShop\PrestaShop\Adapter\Order;
+
+use Order;
+use PrestaShop\PrestaShop\Core\Order\OrderSiblingProviderInterface;
+
+class OrderSiblingProvider implements OrderSiblingProviderInterface
+{
+    /**
+     * {@inheritdoc}
+     */
+    public function getNextOrder(int $orderId): int
+    {
+        $order = new Order($orderId);
+
+        return $order->getNextOrderId();
```

Comment:
Nice finding !

Reply:
it's @jolelievre finding yesterday :slig